src/redux/store.test.ts

```diff
@@ -329,7 +329,7 @@ describe('store state', () => {
         },
         "tokens": Object {
           "error": false,
-          "loading": false,
+          "loading": true,
```

Comment:
is it possible to not update this value? usually the initial state for loading should be false?

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2645/commits/8aa0b731297bb537e31822c32d61d08447f38aa0!